### DISCUSSION TITLE
Suggestion for addition to project

### DISCUSSION THREAD

**Discussion by ducky92** [sanni/cartreader#731] [Ideas & Suggestions]
I just ran across this print and its both open source and was made for this project. Its not mine so I'm not opening a merge request for it but it should be shared. 

https://www.printables.com/model/409926

**Comment by Ancyker** [comment]
_-clears throat-_

#723

**Reply by ducky92** [reply]
I should have looked *face-palm. Thanks @Ancyker 

This discussion post can be closed.
